recommendation/views/images.py

```diff
@@ -0,0 +1,35 @@
+from urllib.parse import urlencode
+
+import requests
+from flask import abort, Blueprint, request, Response, stream_with_context
+
+from recommendation import conf
+
+EMBEDLY_RESIZE = 'https://i.embed.ly/1/display/resize'
+
+
+images = Blueprint('images', __name__)
+
+
+def make_embedly_url(**kwargs):
+    if 'url' not in kwargs:
+        raise RuntimeError()
+    qs = {}
+    for param in ['url', 'width', 'height']:
+        if param in kwargs:
+            qs[param] = kwargs[param][0]
+    qs['key'] = conf.EMBEDLY_API_KEY
+    return '{}?{}'.format(EMBEDLY_RESIZE, urlencode(qs))
+
+
+@images.route('/images')
+def proxy():
+    try:
+        url = make_embedly_url(**request.args)
+    except RuntimeError:
+        abort(400)
+    req = requests.get(url, stream=True)
```

Comment:
IIRC requests doesn't set a default timeout for connections. So I'd put something here manually, and also make sure to catch any exception for either timeouts, SSL errors or anything else that can go wrong with outbound calls:

```
try:
    req = requests.get(url, stream=True, timeout=60.0)  # timeout in seconds
except requests.RequestException:
    abort(400)
```